### PR TITLE
[chore]: Upping usePagination test

### PR DESCRIPTION
**Pull request by OscarBakker**
## Signalen

Before opening a pull request, please ensure:

- Make sure your PR title follows naming conventions: [feat-1234]: name feature
- Double-check your branch is based on `main` and targets `main`
- Pull request has tests (we are going for 100% coverage!)
- Code is well-commented, linted and follows project conventions
- Committed source code is headed by the correct SPDX license expression

Be kind to code reviewers, please try to keep pull requests as small and focused as possible :)

### CODE REVIEW

**Comment by OscarBakker** [comment]
Upping test coverage

Remove first if statement, its typed anyways. The next two if statements are really hard to test. Cannot have an impact on the incidentmaps functioning.

File: src/signals/IncidentMap/components/IncidentMap/usePaginatedIncidents.ts [hunk 1]
```diff
@@ -23,65 +23,77 @@ const usePaginatedIncidents = () => {
...
+      (_, index) => {
+        const qeopageQueryParam = `&geopage=${index + 1}`
+        return fetch(
           `${
             configuration.GEOGRAPHY_PUBLIC_ENDPOINT
-          }?${paginatedIncidents.current.searchParams.toString()}`,
+          }?${paginatedIncidents.current.searchParams.toString()}${qeopageQueryParam}`,
           {
-            method: 'HEAD',
+            method: 'GET',
```
Comment (vdegraaf) [inline src/signals/IncidentMap/components/IncidentMap/usePaginatedIncidents.ts:69]: Why did this become a GET request instead of HEAD?

**Review by vdegraaf — APPROVED** [review]
Approved! Just had a one question.

File: src/signals/IncidentMap/components/IncidentMap/usePaginatedIncidents.ts (hunk 1 above)
Comment (OscarBakker) [reply src/signals/IncidentMap/components/IncidentMap/usePaginatedIncidents.ts:69]: Because it needs to return json, not just head info.

**Pull request merged**